crates/iroha_test_network/src/lib.rs

```diff
@@ -338,76 +396,83 @@ impl NetworkBuilder {
     /// ```
     /// use iroha_test_network::NetworkBuilder;
     ///
-    /// NetworkBuilder::new().with_config(|t| {
+    /// NetworkBuilder::new().with_config_layer(|t| {
     ///     t.write(["logger", "level"], "DEBUG");
     /// });
     /// ```
-    pub fn with_config<F>(mut self, f: F) -> Self
+    pub fn with_config_layer<F>(mut self, f: F) -> Self
```

Comment:
Though the function signature remains unchanged, how are layer priorities determined?

Reply:
They work the same way as `extends` field in config file does:

```toml
extends = ["layer-1.toml", "layer-2.toml", "layer-3.toml"]
```

Layers are merged sequentially, with later ones overriding _conflicting_ parameters from earlier ones.

_(I shall add this into the doc of this method)_

Reply:
Extended method documentation.